pkgs/development/web/bloomrpc/default.nix

```
@@ -0,0 +1,40 @@
+{ lib, fetchurl, appimageTools, pkgs }:
```

```suggestion
{ lib, fetchurl, appimageTools }:
```

I think the construct in extraPkgs does not need this input.